src/components/inputs/TextField.js

```diff
@@ -609,7 +610,7 @@ export default class TextField extends BaseInput {
       const {style, iconSource, iconColor} = rightButtonProps;
```

Comment:
instead of passing all the props included in `rightButtonProps`, pass on the "rest" of the props that were not used specifically. 
```
const {style, iconSource, iconColor, ...others} = rightButtonProps;
```

```
<TouchableOpacity {...others} style={[this.styles.rightButton, style]} onPress={this.onPressRightButton}>

```